Gemfile

```diff
@@ -51,4 +51,5 @@ end
 
 group :repl do
   gem 'pry'
+  gem 'byebug'
```

Comment:
I've removed it from the Gemfile, I still think it's fine in the `.gitignore` file unless we would intend on tracking some sort of byebug history in the repo at any point.